src/app/src/store/store.js

```diff
@@ -1,11 +1,15 @@
 import { configureStore } from '@reduxjs/toolkit';
 
+import api from '../api';
```

Comment:
Since `api/api` is importing from `store/errorSlice`, and we're importing `api/api` into `store/store`, this makes accidental circular references more likely in the future.

Should the RTK Query content be moved into `store`? Is there a better way to organize this?

Reply:
I'm not sure. I like having a top level folder for the api because it will likely accumulate a number of files and they will be referenced from many different places. On the other hand, the endpoints/api are a part of the store and so logically they fit in that folder.

Reply:
Alright, let's keep it for now, but also keep an eye on this.